docs/en/latest/plugins/limit-req.md

```diff
@@ -33,16 +33,30 @@ The `limit-req` Plugin limits the number of requests to your service using the [
-| nodelay           | boolean | False    | false   |                            | If set to `true`, requests within the burst threshold would not be delayed.                                                                                                                                                                                                                                                                                                                           |
-| allow_degradation | boolean | False    | false   |                            | When set to `true` enables Plugin degradation when the Plugin is temporarily unavailable and allows requests to continue.                                                                                                                                                                                                                                                                             |
+| Name                     | Type    | Required | Default     | Valid values                      | Description                                                                                                                                                                                                                                                                                                                                                                                           |
+|--------------------------|---------| -------- |-------------|-----------------------------------| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| conn                     | integer | True     |             | conn > 0                          | Maximum number of concurrent requests allowed. Requests exceeding this ratio (and below `conn` + `burst`) will be delayed (configured by `default_conn_delay`).                                                                                                                                                                                                                                       |
+| burst                    | integer | True     |             | burst >= 0                        | Number of additional concurrent requests allowed to be delayed per second. If the number exceeds this hard limit, they will get rejected immediately.                                                                                                                                                                                                                                                 |
```

Comment:
please check the docs, it has unrelated changes

Reply:
done